### PR TITLE
OSDOCS-3078: Updated some text in the AWS Accounts Limits.

### PR DESCRIPTION
**Pull request by EricPonvelle**
Updated some of the default information in the AWS Accounts Limits.

JIRA: https://issues.redhat.com/browse/OSDOCS-3078

PREVIEW: https://people.redhat.com/eponvell/013122/OSDOCS-3078_AWS-Account-Limits/osd_planning/aws-ccs.html#aws-limits_aws-ccs

### CODE REVIEW

File: modules/aws-limits.adoc [hunk 1]
```diff
@@ -18,13 +18,14 @@ The following table summarizes the AWS components whose limits can impact your a
 |Instance Limits
 |Varies
 |Varies
-|By default, each cluster creates the following instances:
+|At a minimum, each cluster creates the following instances:
 
 * One bootstrap machine, which is removed after installation
 * Three master nodes
-* Three worker nodes
+* Three infrastructure nodes
```
Comment (0kashi) [inline modules/aws-limits.adoc:25]: Can we make this **Two** infrastructure nodes?  2 is the min for a single az and 3 for multi.

Comment (EricPonvelle) [reply modules/aws-limits.adoc]: @0kashi, I changed it to two but do you want me to write that it's two for single and three for multi?

Comment (0kashi) [reply modules/aws-limits.adoc]: I think that's a good idea.

Comment (EricPonvelle) [reply modules/aws-limits.adoc]: Excellent! I updated the PR to include that information. Can you check to make sure it makes sense in the way I presented it?

**Comment by 0kashi** [comment]
lgtm

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**